src/doc/src/reference/unstable.md

```diff
@@ -1022,3 +1060,35 @@ name = "mypackage"
 version = "0.0.1"
 rust-version = "1.42"
 ```
+
+<script>
+(function() {
+    var fragments = {
+        "#edition": "manifest.html#the-edition-field",
+        "#compile-progress": "config.html#termprogresswhen",
+        "#rename-dependency": "specifying-dependencies.html#renaming-dependencies-in-cargotoml",
+        "#alternate-registries": "registries.html",
+        "#offline-mode": "../commands/cargo.html",
+        "#publish-lockfile": "../commands/cargo-package.html",
+        "#default-run": "manifest.html#the-default-run-field",
+        "#cache-messages": "https://github.com/rust-lang/cargo/pull/7450",
+        "#install-upgrade": "../commands/cargo-install.html",
+        "#profile-overrides": "profiles.html#overrides",
+        "#config-profiles": "config.html#profile",
+        "#crate-versions": "https://github.com/rust-lang/cargo/pull/8509",
+        "#features": "features.html#feature-resolver-version-2",
+        "#package-features": "features.html#resolver-version-2-command-line-flags",
+        "#resolver": "resolver.html#resolver-versions",
+    };
```

Comment:
Is this a backwards-compatibility thing? This sems like something that may be hard to keep up-to-date over time perhaps

Reply:
Yea, for example if someone posts a URL on a forum, this will ensure that link doesn't become confusing after the feature gets stabilized.  It might still be confusing, but hopefully people will figure it out.  If these break over time, I don't think it's a big deal.  I can also probably teach the linkchecker to check these, though that's a bit tricky.

Reply:
Ok that's what I figured, and that sounds good to me. That means it's not super critical we keep a ton of stuff here.